cmake/blt-tpl-config.cmake.in

```diff
@@ -0,0 +1,17 @@
+if ("@ENABLE_CUDA@")
+    set(ENABLE_CUDA     "@ENABLE_CUDA@")
+endif()
+
+if ("@ENABLE_HIP@")
+    set(ENABLE_HIP      "@ENABLE_HIP@")
+endif()
+
+if ("@ENABLE_MPI@")
+    set(ENABLE_MPI      "@ENABLE_MPI@")
+endif()
+
+if ("@ENABLE_OPENMP@")
+    set(ENABLE_OPENMP   "@ENABLE_OPENMP@")
+endif()
+
+set (ENABLE_FIND_MPI "@ENABLE_FIND_MPI@")
```

Comment:
```suggestion
# Prefix variables so they don't conflict with non-BLT projects
set(BLT_ENABLE_CUDA        "@ENABLE_CUDA@")
set(BLT_ENABLE_HIP            "@ENABLE_HIP@")
set(BLT_ENABLE_MPI           "@ENABLE_MPI@")
set(BLT_ENABLE_OPENMP   "@ENABLE_OPENMP@")
set(BLT_ENABLE_FIND_MPI  "@ENABLE_FIND_MPI@")
```

Reply:
We should prefix these because these variables were never prefixed with `BLT_` so could clash with downstream non-BLT projects.

This would require some `OR`'s in the BLTSetup*.cmake files. For example:

`if(ENABLE_MPI)` to `if(ENABLE_MPI OR BLT_ENABLE_MPI)`

Reply:
Made these changes